components/table/src/simple-columns/favorite-column.scss

```diff
@@ -0,0 +1,12 @@
+@import './simple-column.scss';
+@import '../../../core/src/style/colors';
+
+.dt-icon {
+  width: 22px;
+  cursor: pointer;
+  fill: $gray-300;
+}
+
```

Comment:
I'm not sure if this will need a hover effect as well, maybe we will need some input from UX here?

Reply:
You were right. I added the hover effect